game/src/main/scala/io/aigar/controller/GameResponse.scala

```diff
@@ -0,0 +1,48 @@
+package io.aigar.controller.response
+
+case class Position(
+  x: Float,
+  y: Float
+)
+case class Direction(
+  x: Float,
+  y: Float
+)
```

Comment:
Should we also add the velocity since we might want to know at which speed the cell is moving? Unless we calculate it with the mass of the cell. It could also depend if we decide that a cell can go to a speed under 100% of its capacity.